exchanges/ftx/ftx.go

```diff
@@ -1378,3 +1379,46 @@ func (f *FTX) SubaccountTransfer(coin currency.Code, source, destination string,
 	}
 	return &resp.Data, nil
 }
+
+// FetchSpotExchangeLimits fetches spot order execution limits
+func (f *FTX) FetchExchangeLimits() ([]order.MinMaxLevel, error) {
+	data, err := f.GetMarkets()
+	if err != nil {
+		return nil, err
+	}
+
+	var limits []order.MinMaxLevel
+	for x := range data {
+		if !data[x].Enabled {
+			continue
+		}
+		var cp currency.Pair
+		var a asset.Item
+		switch data[x].MarketType {
+		case "future":
+			a = asset.Futures
+			cp, err = currency.NewPairFromString(data[x].Name)
+			if err != nil {
+				return nil, err
+			}
+		case "spot":
+			a = asset.Spot
+			cp, err = currency.NewPairFromStrings(data[x].BaseCurrency, data[x].QuoteCurrency)
+			if err != nil {
+				return nil, err
+			}
+		default:
+			return nil, fmt.Errorf("unhandled data type %s, cannot process exchange limit",
+				data[x].MarketType)
+		}
+
+		limits = append(limits, order.MinMaxLevel{
+			Pair:       cp,
+			Asset:      a,
+			StepPrice:  data[x].PriceIncrement,
+			StepAmount: data[x].SizeIncrement,
+			MinAmount:  data[x].SizeIncrement,
```

Comment:
Thoughts on using MinProvideSize here? Seems to be the same as SizeIncrement judging by API output but seems more correct

Reply:
🦅 Will fix! Yeah there are some differences between currencies.